index.html

```diff
@@ -87,6 +87,35 @@
           </div>
         </div>
         <h1 class="heading-48">Component Library</h1>
+        <div class="alert alert--info">
+          <p class="alert__message">
+            The component library is a snapshot of the work found in assets-frontend. Currently assets-frontend is being refactored.
```

Comment:
> This is covered on the home page. Why is it needed on every page?

If people haven't read the homepage or come to a component page directly, I feel its important to emphasise this.